helm/cluster-resources/templates/coredns-np.yaml

```diff
@@ -26,3 +27,33 @@ spec:
   policyTypes:
   - Egress
   - Ingress
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: coredns-cluster-initialisation
+  namespace: kube-system
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+spec:
+  egress:
+  - {}
+  ingress:
+  - ports:
+    - port: 1053
+      protocol: UDP
+    - port: 1053
+      protocol: TCP
+    - port: 53
+      protocol: UDP
+    - port: 53
+      protocol: TCP
+    - port: 9153
```

Comment:
which is this for?

Reply:
it's the `/metrics` endpoint - https://github.com/giantswarm/coredns-app/blob/master/helm/coredns-app/values.yaml#L41-L46

Reply:
> which is this for?

or do you mean the entire new network policy?

Once the adoption of the existing coredns (installed via `kubeadm`) for the new coredns (managed via `appPlatform`) is done, the label-selectors has changed and therefore we need a network policy which match both coredns installations.

This is only requried to make the update for THG for existing clusters as smooth as possible.
For new clusters this isn't an issue and we don't have to take care of this netpol.